components/pricing/CustomServices.tsx

```diff
@@ -0,0 +1,54 @@
+import type { FC } from 'react';
+import Typography from '@mui/material/Typography';
+import Stack from '@mui/material/Stack';
+
+import type { EmptyFunction } from '../../models/common';
+
+import Button from '../common/buttons/Button';
+
+import { CustomServicesContainer } from './styled-components';
+
+interface CustomServicesProps {
+  onOpenDialog?: EmptyFunction;
```

Comment:
In my opinion `onOpenDialog` can't be optional

Reply:
'+